### PR TITLE
18-220 changes

### PR DESCRIPTION
**Pull request by emilyszabo27**
# Summary of Changes

- Please indicate the classes you are modifying:
  - 18-100
  - ...

# Checklist:

- [ ] I have added my entries to the `README.md` table of contents in the root directory as links.

Make sure that your contributions:

- [ ] Are accurate and will not mislead the reader
- [ ] Do not encourage or cause academic integrity violations
- [ ] Do not offend professors or unfairly criticize the university

### CODE REVIEW

File: ece_core/18220.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 | Category | Difficulty |
 |:-:       | :-:        |
-| HW       | 5          |
+| HW       | 3          |
```
Comment (mikinty) [inline ece_core/18220.md:5]: I don't know how we want to do this in the future, but I think we should get a distribution of what people think of difficulties...if you have any ideas of how to do that please lmk

I would like to keep HW at 5 if possible, I don't think it's a 3 which signals it's easy. 6 for a lab is much more reasonable I agree.

File: ece_core/18220.md [hunk 2]
```diff
@@ -64,14 +57,11 @@ effective way to teach students about how to work with circuits. There are labs
 - Do the lab too slowly
 - Break lab rules, i.e. making a mess, not turning off equipment, eating in class
 
-Just follow the rules, be nice to your TAs, and you will get 100%.
+The labs can become very confusing if you don't do adequate prep work. Definitely make an effort to have the prelab completed by the time you even walk into the lab: this will help you get off to a great start. Something else that isn't necessary but definitely also helps is reading the entire lab before going in to work on the lab. This helps you gain an overview of what you'll be doing during the lab so you can have a better understanding of your goals for the lab.
 
-When making circuits, people often just string together whatever they can, and just pray it works. What you should do instead, is treat your circuit like code you write in software. Would you write messy code? Would you write code that is not possible to debug, i.e. not have anywhere to insert `print` statements or set breakpoints? If you think about making "good circuits" as you do lab, you will make you and your TA's lives much better.
```
Comment (mikinty) [inline ece_core/18220.md:69]: I don't think this needs to be deleted

File: ece_core/18220.md [hunk 3]
```diff
@@ -64,14 +57,11 @@ effective way to teach students about how to work with circuits. There are labs
...
 
-- Make your circuits as neat as possible
-- Cut wires long enough so you don't have to cram everything in a small space
-- Likewise, don't make wires ridiculously long so "you can always extend your connection" You will have a hot spaghetti mess on your board
-- You have an oscilloscope, which means you can literally probe in anywhere in your circuit and measure what's going on. This is basically an irl super debugger.
```
Comment (mikinty) [inline ece_core/18220.md:74]: Are these not useful tips?

File: ece_core/18220.md [hunk 4]
```diff
@@ -85,11 +75,16 @@ Therefore, a reasonable study plan is to
...
 
+As mentioned before, the exams are a time crunch. Keep this in mind before you go into the exam room.
+
+The final exam is not significantly different than the midterm exams. There is a makeup exam the week before finals week, and taking that exam is a great way to study for the final.
+
 
 ## How to do well
 
-- Know how to apply KCL to any circuit, no matter how complicated.
-- Keep a list of equations/properties as you take the course, as they will help you study for exams in a more condensed way
```
Comment (mikinty) [inline ece_core/18220.md:92]: Are these also not useful?

File: ece_core/18220.md [hunk 5]
```diff
@@ -27,32 +27,25 @@ storage systems work, and how devices are connected and powered.
...
-for every topic. While this is not necessarily bad, the concepts taught in class
-may not necessarily correspond as closely to the homeworks as you think, which
-then makes people feel like the lectures are not that useful. I think this is
-done intentionally to challenge students, but I do feel like certain topics in
-the class could be taught more comprehensively, such as
-
-- Second-order circuits
-- Feedback systems
-- Bode plots
-- Small signal
```
Comment (mikinty) [inline ece_core/18220.md:55]: Is this not useful?

**Review by mikinty — CHANGES_REQUESTED** [review]
I think too much is deleted from this note, unless things have changed drastically in the last 2 years

**Comment by emilyszabo27** [comment]
I put in another pull request because I'm not sure how to edit this one. Feel free to dismiss this one and only consider the new pull request, and let me know your thoughts on the new pull request.

**Pull request merged**